int16 id
float32 dist_long
float32 dist_lat
float32 vrel_long
int8 dyn_prop
float32 vrel_lat
float32 rcs
float32 dist_long_rms
float32 vrel_long_rms
float32 dist_lat_rms
float32 vrel_lat_rms
float32 arel_lat_rms
float32 areal_long_rms
float32 orientation_rms
int8 meas_state
float32 prob_of_exist
float32 arel_long
float32 arel_lat
int8 obj_class
float32 orientation_angel
float32 obj_len
float32 obj_width